### ISSUE TITLE
Flutter doctor complains about Flutter Extension for vscode even when it is installed

### ISSUE THREAD

**Issue by degroote22** [flutter/flutter#19292]
## Logs
```
$ ~/development/flutter/bin/flutter doctor -v
[✓] Flutter (Channel beta, v0.5.1, on Mac OS X 10.13.6 17G54a, locale pt-BR)
    • Flutter version 0.5.1 at /Users/lucasdeavilamartins/development/flutter
    • Framework revision c7ea3ca377 (6 weeks ago), 2018-05-29 21:07:33 +0200
    • Engine revision 1ed25ca7b7
    • Dart version 2.0.0-dev.58.0.flutter-f981f09760

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.0)
    • Android SDK at /Users/lucasdeavilamartins/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 9.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 9.4.1, Build version 9F2000
    • ios-deploy 1.9.2
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 25.0.1
    • Dart plugin version 173.4700
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)

[!] VS Code (version 1.25.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected devices (1 available)
    • iPhone 5s • 6A91E1F6-2B7C-4014-9597-CFE175290A6A • ios • iOS 11.4 (simulator)

! Doctor found issues in 1 category.
```

```
$~/development/flutter/bin/flutter analyze
Analyzing flutter...
No issues found! (ran in 18.1s)
```

The extension is active and I can use it to create a new project, for instance.
Doctor is complaining about the extension when it shouldn't.
Thanks

**Comment by zoechi** [comment]
@DanTup

**Comment by DanTup** [comment]
This is fixed by #18515 but it hasn't made the beta channel yet. It's safe to ignore that warning for now; it won't affect your use of Flutter or VS Code.
